### ISSUE TITLE
[osx request] XCode 10 beta image

### ISSUE THREAD

**Issue by rubenvb** [travis-ci/travis-ci#9730]
Xcode 10 beta was released: https://developer.apple.com/xcode/

**Comment by djones6** [comment]
Swift 4.2 snapshots later than 2018-06-04-a now require Xcode 10 beta, so we are currently unable to run CI with the latest Swift snapshots on macOS.

**Comment by ikesyo** [comment]
Xcode 10 beta 3 is out now.

**Comment by ikesyo** [comment]
Xcode 10 beta 4 is out now.

**Comment by fssilva** [comment]
Would be nice to see `Xcode 10` image available on Travis.

**Comment by rplankenhorn** [comment]
Is there anyone working on this on the Travis team?  Any progress updates?

**Comment by mxcl** [comment]
Due to the lack of any responses here, I took a look into Circle CI, but they also don’t have an image, AND also are not respecting their community enough to even update them on progress:

https://discuss.circleci.com/t/xcode-10-beta-vm/23238/3

**Comment by rplankenhorn** [comment]
@mxcl Yeah this is really unfortunate for a paid product.  Doesn't seem like there is any interest to support Xcode 10.

I would also be interested in a CI environment that supports Xcode 10.  Right now, my plan is to do manual builds using a second laptop and upload to Fabric.  Just unfortunate that I have to resort to this.

**Comment by rplankenhorn** [comment]
@mxcl I found this issue about Xcode 9 Beta:  https://github.com/travis-ci/travis-ci/issues/7865.

I tried emailing @solarce but it bounced back.  Going to take a few more people from that issue to see if we can get any kind of communication regarding this issue.

@courteouselk @emdantrim

**Comment by cotsog** [comment]
Thanks everyone for your input and I'm sorry for the radio silence on our end.

We are discussing this internally and will have more information late next week.

Thank you for your patience!

**Comment by turbolent** [comment]
@cotsog Any update on this?

**Comment by rplankenhorn** [comment]
@cotsog Is there an update?

**Comment by rplankenhorn** [comment]
Looks like CircleCI has released their VM:  https://discuss.circleci.com/t/xcode-10-0-beta-5-released/24035
